<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#login{background-image:url(:/imgs/imgs/loginBk.png)}</string>
  </property>
  <widget class="QLabel" name="pwdLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>337</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>335</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(119, 226, 137);
background-color: rgba(255, 255, 255, 180);
border:none;
border-radius:3px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>83</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color:rgba(255, 255, 255, 0);
border:0px;

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>295</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 180);
border:none;
border-radius:3px;
color:rgb(119, 226, 137);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logonButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>83</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border:0px;
background-color:rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>403</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(186, 186, 186);</string>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
